docs/guides/chip_tool_guide.md

```diff
@@ -778,25 +841,74 @@ following command:
+    [Commissioning into a Thread network over Bluetooth LE](#commissioning-into-a-thread-network-over-bluetooth-le).
+-   `ble-wifi` - For Wi-Fi commissioning; described under
+    [Commissioning into a Wi-Fi network over Bluetooth LE](#commissioning-into-a-wi-fi-network-over-bluetooth-le)
+-   `onnetwork` - For commissioning the device when it is already present in an
+    IP network; described under
+    [Commissioning with setup PIN code](#commissioning-with-setup-pin-code)
 
-To list all `pairing` sub-commands, run the following command:
+To list all `pairing` sub-commands and commissioning methods, run the following
+command:
 
 ```
 $ ./chip-tool pairing
 ```
 
+#### Attestation-related flags
+
+The `pairing` commissioning command can be run with several flags that allow you
+to modify attestation-related settings:
+
+-   `--paa-trust-store-path` - Use to provide path to the directory that
+    contains the information about Product Attestation Authority (PAA)
+    certificate. The path can be absolute or relative to the current working
+    directory. With this flag, the CHIP Tool looks for the PAA certificate that
+    matches the PAI and the DAC certificates programmed on the device. Without
+    this flag, the CHIP Tool uses the test PAA certificate.
+
+-   `--cd-trust-store-path` - Use to provide path to the directory that contains
+    the information about Certificate Declaration. The path can be absolute or
+    relative to the current working directory. With this flag, the CHIP tool
+    looks for the CD that matches other attestation data on the device. Without
+    this flag, the CHIP Tool uses the test CD.
+
+-   `--only-allow-trusted-cd-keys` - Use to only allow trusted CD verifying keys
+    (disallow test keys). If the flag is not provided or it is provided with the
+    value `0` ("false"), untrusted CD verifying keys are allowed. If it is
+    provided with the value `1` ("true", `--only-allow-trusted-cd-keys 1`), test
+    keys are disallowed and CD used for attestation will not be accepted. This
+    setting is parsing CD for an `int` enumeration that sets the test status of
+    CD.
+
+-   `--bypass-attestation-verifier` - Use to bypass the attestation verifier. If
+    the flag is not provided or it is provided with the value `0` ("false"), the
+    attestation verifier is not bypassed. If it is provided with the value `1`
+    ("true", `--bypass-attestation-verifier 1`), the commissioning will continue
+    in case of the attestation verification failure. The failure can be caused
+    by errors in Certification Declaration, the PAA or the PAI certificates, or
+    in the Device Attestation Certificate. This option can be useful is you want
+    to quickly commission a device with the PAI and the DAC certificates that
+    are based on an unknown PAI certificate, or when using a CD to which we
+    don't have access (for example it is part of an officially certified
+    device).
```

Comment:
I don't understand why we would need "access" to the CD.  This part does not make sense.

Reply:
I think it is connected with this comment:
https://github.com/project-chip/connectedhomeip/pull/28358/files/232b0248a7307e54f605112a4de8f5edbc77dabe#r1277781710
bypass-attestation should be used if we don't have access to Certification Declaration signing key.

Reply:
The "CD signing key" is not the same thing as "the CD".